acquisitions/src/main/resources/thunderjet/mod-invoice/reusable/create-invoice-line.feature

```diff
@@ -20,7 +19,6 @@ Feature: Create invoice line
     * set invoiceLine.subTotal = total
     * set invoiceLine.fundDistributions[0].expenseClassId = expenseClassId
     * set invoiceLine.releaseEncumbrance = releaseEncumbrance
-    * set invoiceLine.adjustments = adjustments
```

Comment:
before to merge I think on time search for this usage in other places. if latest scenario used this, it will fail